src/EncoderProvider.php

```diff
@@ -19,7 +19,7 @@
 
 final class EncoderProvider implements ResetInterface
 {
-    private const ENCODINGS = [
+    public const ENCODINGS = [
```

Comment:
I can only agree that it does not need changing for tests only having a public modifier there wouldn't cause any trouble I guess and it is convenient having it this way as it can reused at several places.

Reply:
I prefer to reduce the public API to minimize changes that break backward compatibility in the future. But I don't mind if this constant is part of the public API.